src/js/components/Peak/PeakStatistics.tsx

```diff
@@ -39,12 +49,32 @@ const PeakStatistics = ({ peakId }: { peakId: string }) => {
             </span>
           </>
         ) : (
-          <span className={styles.noFirstConqueror}>-</span>
+          <span className={styles.noConqueror}>-</span>
         )}
-      </div>
+      </p>
+      <p className={styles.wrapper}>
+        Ostatni zdobywca:
+        {lastConqueror ? (
+          <>
+            <img
```

Comment:
Skoro ma być to okrągła ikona ze zdjęciem to lepiej użyć komponentu <Icon> tak żebyśmy mieli spójność